modin/pandas/test/dataframe/test_map_metadata.py

```diff
@@ -1289,7 +1289,23 @@ def test_update(data, other_data):
     df_equals(modin_df, pandas_df)
 
     with pytest.raises(ValueError):
+        pandas_df.update(other_pandas_df, errors="raise")
+
+    if Engine.get() == "Python":
+        with pytest.raises(ValueError):
+            modin_df.update(other_modin_df, errors="raise")
+    else:
+        # We expect a ValueError because other_modin_df and modin_df have
+        # non-null values at some of the same locations. When using engines
+        # other than python, exception occurs in remot tasks, but the main
+        # thread doesn't get the exception until it tries to materialize the
+        # remote functions' results. We use ._to_pandas() to materialize the
+        # remote functions' results.
+        # TODO(https://github.com/modin-project/modin/issues/3966): Use a more
```

Comment:
Can't we use `eval_general` for this purpose? https://github.com/modin-project/modin/blob/312c443d88f5b548ee033f223b4c2363701c4cdb/modin/pandas/test/utils.py#L708

Reply:
Done.